src/main/java/uk/gov/companieshouse/search/api/service/search/impl/dissolved/DissolvedSearchRequestService.java

```diff
@@ -41,8 +42,12 @@
     private static final String RESULT_FOUND = "A result has been found";
     private static final String SEARCH_HITS = "searchHits";
 
-    public DissolvedSearchResults getSearchResults(String companyName, String requestId) throws SearchException {
+    public DissolvedSearchResults getSearchResults(String companyName, String searchBefore, String searchAfter,
```

Comment:
Can fix sonar lint issue by updating to be ` public DissolvedSearchResults<DissolvedCompany> getSearchResults`

Reply:
Good point, will do that